config/cards.yml

```diff
@@ -126,23 +125,24 @@
 
     - title: "Web Sign-In"
       url: "/videos/learn-identity-series/web-sign-in"
-
+      
 - id: "extensibility"
   title: "Extensibility"
   icon: "icon-budicon-173"
   color: "red"
   description: "Customize Auth0's standard feature set."
   url: "/extend-integrate"
   articles:
+  
+    - title: "Marketplace"
+      url: "https://cybertronn.herokuapp.com/"
```

Comment:
Yes, it's noted in the comments for the PR. :) I guess I can go ahead and change it to the prod URL, but I had made it staging since the staging site is where the Heroku build is, so Product Marketing could see the intent and sign off first. (This PR is still in process. We're still weighing microsites/card placement, etc.)

Reply:
🤦 sorry, totally missed that!

Best staging to use, then, is `https://marketplace.stage.auth0.com/`

Reply:
Done.